### PR TITLE
Add doc(sdk-experiments): add final doc to  readme file

### LINKED ISSUE #28215: Add doc(sdk-experiments): add final doc to readme file
### Parent Issue
#27539 

### Task

Add doc to @dotcms/experiments to publish to NPM

### Proposed Objective

Documentation

### Proposed Priority

Priority 2 - Important

### Acceptance Criteria

_No response_

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

_No response_

### Assumptions & Initiation Needs

_No response_

### Quality Assurance Notes & Workarounds

_No response_

### Sub-Tasks & Estimates

_No response_

### PR DESCRIPTION
**Pull request by oidacra**
### Proposed Changes
*  add final doc to  readme file

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

### CODE REVIEW

File: core-web/libs/sdk/experiments/README.md [hunk 1]
```diff
@@ -1,32 +1,68 @@
 # @dotcms/experiments
 
-`@dotcms/experiments` is the official dotCMS JavaScript library to helps add A/B Testing to Single Page Applications (SPAs). It handles user assignment to different variants of a page and tracks.
+`@dotcms/experiments` is the official dotCMS JavaScript library that helps add A/B testing to Single Page Applications (SPAs). It handles user assignments to different variants of a page and tracks their interactions.
```
Comment (fmontes) [inline core-web/libs/sdk/experiments/README.md:3]:
```suggestion
`@dotcms/experiments` is the official dotCMS JavaScript library that helps add A/B testing to your webapps. It handle user assignments to different variants of a page and tracks their interactions.
```

File: core-web/libs/sdk/experiments/README.md [hunk 2]
```diff
@@ -1,32 +1,68 @@
...
 
 ## Features
 
--   WIP
+This library facilitates the integration of A/B testing into your SPA by providing robust features for managing user experiences and tracking interactions:
```
Comment (fmontes) [inline core-web/libs/sdk/experiments/README.md:7]:
```suggestion

```

You already said this in the intro.

File: core-web/libs/sdk/experiments/README.md
Comment (fmontes) [inline core-web/libs/sdk/experiments/README.md:1]: Explain very shortly how the dotcms A/B testing work, mention assignment, pagesviews

File: core-web/libs/sdk/experiments/README.md [hunk 3]
```diff
@@ -1,32 +1,68 @@
...
 yarn add @dotcms/experiments
 ```
 
-## Usage
 
-WIP
+## Components
+
+### `DotExperimentsProvider`
+This component utilizes React's Context API to provide DotExperiments instances to its descendants, facilitating access to A/B testing features throughout your application.
```
Comment (fmontes) [inline core-web/libs/sdk/experiments/README.md:31]: > facilitating access to A/B testing features throughout your application

Double check this.

**Review by fmontes — APPROVED** [review]

File: core-web/libs/sdk/experiments/README.md (hunk 1 above)
Comment (oidacra) [reply core-web/libs/sdk/experiments/README.md:3]: changed.

File: core-web/libs/sdk/experiments/README.md (hunk 2 above)
Comment (oidacra) [reply core-web/libs/sdk/experiments/README.md:7]: deleted.

File: core-web/libs/sdk/experiments/README.md
Comment (oidacra) [reply core-web/libs/sdk/experiments/README.md:1]: added

**Review by hmoreras — APPROVED** [review]

**Pull request merged**